src/Components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import '../Themes/Button.css'; 
+export default class Button extends React.Component{
+  render() {
+    return(
+      <button onClick="" className="button" type={this.props.type}>{this.props.name}</button>
```

Comment:
Add a prop `onClickHandle` to handle the `onClick` events.
```suggestion
      <button onClick={this.props.onClickHandle} className="button" type={this.props.type}>{this.props.name}</button>
```